docs/hello.rst

```diff
@@ -65,16 +63,35 @@ You can also reuse the package management system, Python IDEs, and existing Pyth
 Portability
 -----------------
 
-Taichi supports both CPUs and NVIDIA GPUs.
+Taichi code can run on CPUs or GPUs. Depends on your platform:
```

Comment:
Depends on your platform -> Initialize Taichi according to your hardware platform